<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditorDialog</class>
 <widget class="QDialog" name="ItemEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblCode">
     <property name="text">
      <string>&amp;Код</string>
     </property>
     <property name="buddy">
      <cstring>edtCode</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="edtCode"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>&amp;Наименование</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="edtName"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Период</string>
     </property>
     <property name="buddy">
      <cstring>edtPeriod</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QSpinBox" name="edtPeriod">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>7</number>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="CInDocTableView" name="tblItems"/>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtPeriod</tabstop>
  <tabstop>tblItems</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
